quality-assurance/user-acceptance-testing.md

```diff
@@ -0,0 +1,41 @@
+## What is User Acceptance Testing
+
+UAT means testing software against its business requirements i.e. does it solve the business problem as intended?
```

Comment:
need to state that these have been clearly stated in the Epic/features and stories in terms of capturing the requirements.
This needs to cover things like key business rules and logic and the user journey flows that are important for the business.
Diagrams and flow charts are your friend

Reply:
added stuff about business rules